package proto_ff;

// calm 自定义的fieldoption都在该文件中
import "yd_fieldoptions.proto";


//客户端和服务器之间的协议ID
enum ClientServerCmd
{
  //登录 1 - 100
  CLIENT_LOGIN_REQ = 1;			//登录请求
  CLIENT_LOGIN_RSP = 2;			//返回登录结果

  CLIENT_CREATE_ROLE_REQ = 3;		//创角请求
  CLIENT_CREATE_ROLE_RSP = 4;		//返回创角结果

  CLIENT_QUEUE_POS_REQ = 5;		//排队位置请求
  CLIENT_QUEUE_POS_RSP = 6;		//返回排队位置


  CLIENT_QUEUE_CANCEL_REQ = 7;	//取消排队请求
  CLIENT_QUEUE_CANCEL_RSP = 8;	//返回取消排队结果


  CLIENT_RECONNECT_REQ = 9;		//断线重连请求
  CLIENT_RECONNECT_RSP = 10;		//返回重连结果

  CLIENT_ENTER_GAME_REQ  = 11;	//进入游戏请求
  CLIENT_ENTER_GAME_RSP  = 12;	//返回进入游戏结果

  CLIENT_RET_ROLE_LIST_REQ = 13;	//请求返回角色列表界面
  CLIENT_RET_ROLE_LIST_RSP = 14;	//返回角色列表界面 的结果

  CLIENT_PING_REQ	= 15;			//客户端ping消息
  CLIENT_PING_RSP	= 16;			//服务器返回 ping 结果



  //视野 101 - 150
  NOTIFY_CLIENT_CREATURE_CREATE = 101;		//通知客户端创建生物
  NOTIFY_CLIENT_CREATURE_DESTORY = 102;		//通知客户端销毁生物
  BROAD_CHG_NAME = 103;						//广播角色改名


  //场景 151 - 200
  NOTIFY_CLIENT_LOAD_MAP	= 151;				//通知客户端加载场景
  NOTIFY_CLIENT_TRANS_SCENE_RSP = 152;		//通知客户端切换场景结果
  CLIENT_LOAD_MAP_FINISH = 153;				//客户端加载地图完成
  CLIENT_SCENE_TRANS_REQ = 154;				//客户端请求场景传送
  NOTIFY_SCENE_CORPSE_INFO_RSP = 155;			//场景尸体信息
  NOTIFY_SCENE_DRIVE_RSP = 156;				//场景驱离信息

  //背包 200-210
  CLIENT_PACKAGE_INFO_REQ = 200;	//请求背包信息
  CLIENT_PACKAGE_INFO_RSP = 201;	//返回背包信息

  CLIENT_PACKAGE_SORT_REQ = 202;	//整理背包请求
  CLIENT_PACKAGE_SORT_RSP = 203;	//整理背包返回

  CLIENT_STORAGE_OPERATE_REQ = 204; //仓库和背包之间的操作(物品移进移出)请求
  CLIENT_STORAGE_OPERATE_RSP = 205; //仓库和背包之间的操作(物品移进移出)返回

  NOTIFY_PACKAGE_UPDATE = 206;	//通知背包更新

  CLIENT_TO_LOGIC_PACKAGE_USE = 207;		//使用物品
  LOGIC_TO_CLIENT_PACKAGE_USE = 208;


  //玩家 250-300
  CLIENT_PLAYER_INFO_RSP	= 251;	 //返回 玩家信息
  CREATURE_ATTR_SYN = 252;		 //生物属性同步
  CREATURE_ATTR_BROAD = 253;		 //生物属性广播
  CLIENT_SHOW_INFO_RSP = 254;		 //通知客户端显示信息(主要用于客户端提示、飘字等显示用)
  NOTIFY_FACADE_CAST_RSP = 255;	 //广播外观数据
  CLIENT_CHG_NAME_REQ = 256;		 //请求 角色改名
  CLIENT_CHG_NAME_RSP = 257;		 //返回 角色改名 结果
  CREATURE_STATE_BROAD = 258;		 //生物状态广播


  // 打坐
  CLIENT_TO_LOGIC_PLAYER_SEAT_REQ					= 259;	 //玩家请求打坐
  LOGIC_TO_CLIENT_SEAT_RSP								= 260;	// 打坐返回
  LOGIC_TO_CLIENT_SEAT_EXP_RSP						= 261;	// 打坐没分钟经验返回
  CLIENT_TO_LOGIC_LOGIN_HANGUP_DATA_REQ		= 262;	 //玩家登陆获取挂机数据
  LOGIC_TO_CLIENT_LOGIN_HANGUP_DATA_RSP		= 263;	// 获取上线挂机数据返回
  CLIENT_TO_LOGIC_GET_HANGUP_REWARD_REQ   = 264;  //获取挂机奖励
  LOGIC_TO_CLIENT_GET_HANGUP_REWARD_RSP   = 265;  //获取挂机奖励返回

  //GM模块
  CLIENT_GM_REQ = 270;		//GM请求
  CLIENT_GM_RSP = 271;		//GM返回

  //养成影响外观
  NOTIFY_GROW_RSP = 281;				//养成数据返回
  CLIENT_GROW_LVUP_REQ = 282;			//养成升星请求
  CLIENT_GROW_LVUP_RSP = 283;			//养成升星返回
  CLIENT_GROW_FACADE_DRESS_REQ = 284;	//养成外观穿戴请求
  CLIENT_GROW_FACADE_DRESS_RSP = 285; //养成外观穿戴返回
  CLIENT_GROW_FACADE_UNDRESS_REQ = 286;	//养成外观卸下请求
  CLIENT_GROW_FACADE_UNDRESS_RSP = 287;	//养成外观卸下返回

  //装备养成
  CLIENT_EQUIP_DRESS_REQ = 300;			//装备穿戴请求
  CLIENT_EQUIP_DRESS_RSP = 301;			//装备穿戴返回
  CLIENT_EQUIP_UNDRESS_REQ = 302;			//装备卸下请求
  CLIENT_EQUIP_UNDRESS_RSP = 303;			//装备卸下返回
  NOTIFY_EQUIP_INFO_RSP 	= 304;			//装备更新通知

  //任务模块 350 - 400
  CLIENT_TO_SERVER_MISSION_BEGIN			= 350;
  CLIENT_TO_LOGIC_QUERY_MiSSIONLIST		= 351;	//请求任务列表
  LOGIC_TO_CLIENT_QUERY_MiSSIONLIST		= 352;      // 请求任务列表返回
  CLIENT_TO_LOGIC_ACCEPTMISSION				= 353;  //请求接取任务(停用)
  LOGIC_TO_CLIENT_ACCEPTMISSION				= 354;		// 接收任务返回	(停用)
  CLIENT_TO_LOGIC_ABANDONMISSION			= 355;  //请求放弃任务
  LOGIC_TO_CLIENT_ABANDONMISSION			= 356;		// 放弃任务返回
  CLIENT_TO_LOGIC_SUBMITMISSION				= 357;  //请求提交任务
  LOGIC_TO_CLIENT_SUBMITMISSION				= 358;		// 提交任务返回
  CLIENT_TO_LOGIC_TALKWITHNPC					= 359;  //请求对话请求
  CLIENT_TO_LOGIC_SUBMITMISSIONGOODS		= 361;	//提交任务物品
  LOGIC_TO_CLIENT_SUBMITMISSIONGOODS			= 362;		// 提交任务物品返回
  CLIENT_TO_LOGIC_NPC_ACCEPTMISSION		= 363;	//NPC接取任务
  LOGIC_TO_CLIENT_NPC_ACCEPTMISSION			= 364;		// NPC接取任务返回
  CLIENT_TO_LOGIC_MISSION_ONEKEY_FINISH	= 365;	//一键完成任务
  LOGIC_TO_CLIENT_MISSION_ONEKEY_FINISH		= 366;		//一键完成任务
  CLIENT_TO_LOGIC_MISSION_ONCE_FINISH		= 367;	//立即完成任务
  LOGIC_TO_CLIENT_MISSION_ONCE_FINISH			= 368;		//立即完成任务
  LOGIC_TO_CLIENT_UPDATEMISSIONSTATUSNOTIFY	= 370;		// 任务状态更新
  LOGIC_TO_CLIENT_UPDDATE_DY_ACCEPT_NUM		= 371;		// 更新动态任务已经接取次数
  LOGIC_TO_CLIENT_RECEIVEMISSIONNOTIFY		= 372;      // 接取任务通知
  LOGIC_TO_CLIENT_GCDELETEMISSIONNOTIFY		= 373;      // 删除已接任务通知
  CLIENT_TO_SERVER_MISSION_END			= 400;


  //聊天模块 430 - 450
  CLIENT_SERVER_CHAT_BEGIN						= 430;
  CLIENT_TO_LOGIC_CHAT_MSG_REQ					= 431;	//请求聊天
  LOGIC_TO_CLIENT_CHAT_MSG_RSP					= 432;	// 聊天返回
  CLIENT_TO_CENTER_ASK_OFFLINE_MSG_REQ			= 433;	//请求好友聊天离线数据
  CENTER_TO_CLIENT_ASK_OFFLINE_MSG_RSP			= 434;	//获取好友离线消息
  CLIENT_TO_CENTER_INTRVAL_ASK_OFFLINE_MSG_REQ	= 435;	//分段请求聊天离线数据
  CENTER_TO_CLIENT_INTRVAL_ASK_OFFLINE_MSG_RSP	= 436;	//分段获取好友离线消息
  LOGIC_TO_CLIENT_SYSTEM_CHAT_MSG_NOTIFY			= 437;	// 系统消息返回
  SERVER_TO_CLIENT_SHOW_MESSAGE_NOTIFY			= 438;	//显示消息
  CLIENT_SERVER_CHAT_END							= 450;

  //好友模块 460 - 500
  CLIENT_TO_SERVER_FRIEND_BEGIN		 = 460;	 //BEGIN   FRIENED
  CLIENT_TO_CENTER_RELATION_DATA_REQ       = 461;	 //好友数据请求
  CENTER_TO_CLIENT_RELATION_DATA_RSP			= 462;  //好友数据返回
  CLIENT_TO_CENTER_FRIEND_APPLY_REQ		 = 463;  //申请添加好友
  CENTER_TO_CLIENT_FRIEND_APPLY_RSP			= 464;  //申请添加好友返回消息
  CLIENT_TO_CENTER_AGREE_APPLY_REQ	     = 465;  //同意添加好友
  CENTER_TO_CLIENT_AGREE_APPLY_RSP			= 466;  //同意添加好友返回消息
  CENTER_TO_CLIENT_SYNC_RELATION_NOTIFY			= 467;  //增加关系同步消息
  CENTER_TO_CLIENT_RELATION_ONLINE_NOTIFY		= 468;  //好友在线状态改变
  CLIENT_TO_CENTER_ADD_BLACK_REQ	         = 469;  //加黑名单请求
  CENTER_TO_CLIENT_ADD_BLACK_RSP				= 470;  //添加黑名单返回消息
  CLIENT_TO_CENTER_DELETE_RELATION_REQ	 = 471;  //删除关系（好友，黑名单，，，）
  CENTER_TO_CLIENT_DELETE_RELATION_RSP		= 472;  //删除关系返回消息
  CENTER_TO_CLIENT_SYNC_DELRELATION_NOTIFY		= 473;  //删除关系同步消息
  CLIENT_TO_CENTER_RECOMMEND_FRIEND_REQ	 = 474;	 //好友推荐
  CENTER_TO_CLIENT_RECOMMEND_FRIEND_RSP		= 475;  //好友推荐返回消息
  CLIENT_TO_CENTER_ADD_SHIELD_RELATION_REQ = 476;  //添加发言屏蔽
  CENTER_TO_CLIENT_ADD_SHIELD_RELATION_RSP	= 477;  //添加发言屏蔽返回消息
  //好友送礼
  CLIENT_TO_CENTER_GIFT_GIVING_RECORD_REQ	 = 490;  //送礼记录请求
  CENTER_TO_CLIENT_GIFT_GIVING_RECORD_RSP		= 491;  //送礼记录返回消息
  CLIENT_TO_CENTER_GIVING_GIFT_REQ		 = 492;	 //送礼请求
  CENTER_TO_CLIENT_GIVING_GIFT_RSP			= 493;	//送礼返回消息
  CENTER_TO_CLIENT_GIVING_GIFT_NOTICE		= 494;  //收到送礼消息
  CLIENT_TO_CENTER_THANKS_GIVING_GIFT	 = 495;	 //感谢请求
  CLIENT_TO_SERVER_FRIEND_END			 = 500;	 //END    FRIEND

  //查询 500-510
  CLIENT_TO_CENTER_QUERY_BEGIN				= 501;
  CLIENT_TO_CENTER_CHARACTER_DETAIL_DATA		= 502;	//查看玩家详细数据
  CENTER_TO_CLIENT_CHARACTER_DETAIL_DATA		= 503;		//查看玩家详细数据返回
  CLIENT_TO_CENTER_CHARACTER_MINI_DATA		= 504;	//查看玩家简要数据
  CENTER_TO_CLIENT_CHARACTER_MINI_DATA		= 505;		//查看玩家简要数据返回
  CLIENT_TO_CENTER_CHAR_FUNC_FIGHT_DATA		= 506;	//查看角色功能点战力数据
  CENTER_TO_CLIENT_CHAR_FUNC_FIGHT_DATA		= 507;		//查看角色功能点战力数据返回
  CLIENT_TO_CENTER_QUERY_END					= 510;


  //邮件模块 560-579
  CLIENT_SERVER_MAIL_BEGIN									= 560;
  CLIENT_TO_CENTER_GETMAILITEMLIST_REQ			= 561;
  SERVER_TO_CLIENT_GETMAILITEMLIST_RSP			= 562;
  CLIENT_TO_CENTER_ASKMAIL_REQ							= 563;
  SERVER_TO_CLIENT_ASKMAIL_RSP							= 564;
  CLIENT_TO_CENTER_DELETEMAIL_REQ						= 565;
  SERVER_TO_CLIENT_DELETEMAIL_RSP						= 566;
  CLIENT_TO_CENTER_GETALLMAILATTACHMENT_REQ	= 567;
  SERVER_TO_CLIENT_GETALLMAILATTACHMENT_RSP = 568;
  CLIENT_TO_CENTER_GETMAILATTACHMENT_REQ		= 570;
  SERVER_TO_CLIENT_GETMAILATTACHMENT_RSP		= 569;
  SERVER_TO_CLIENT_UNREADMAIL_NOTIFY				= 571;
  SERVER_TO_CLIENT_NEWMAIL_NOTIFY						= 572;



  CLIENT_SERVER_MAIL_END										= 579;

  //功能解锁 580-589
  SERVER_TO_CLIENT_FUNCTIONUNLOCK_NOTIFY		= 580;
  SERVER_TO_CLIENT_FUNCTIONUNLOCK_INFO			= 581;
  CLIENT_TO_SERVER_FUNCTIONUNLOCK_GET_REWARD_REQ		= 582;
  SERVER_TO_CLIENT_FUNCTIONUNLOCK_GET_REWARD_RSP		= 583;


  //移动 601 - 650
  CLIENT_MOVE_REQ = 601;				//请求移动
  MOVE_BROAD_RSP = 602;				//生物移动广播
  MOVE_TELEPORT_RSP = 603;			//瞬移通知

  //死亡复活 651 - 700
  PLAYER_RELIVE_NOTIFY = 651;			//通知客户端死亡复活
  PLAYER_RELIVE_REQ = 652;			//请求复活
  PLAYER_RELIVE_RSP = 653;			//返回 复活结果

  //技能 700-750
  SKILL_USE_REQ	= 701;				//请求 使用技能
  SKILL_TARGET_SYNC_REQ = 702;		//同步 技能目标
  SKILL_USE_RESULT_BROAD = 703;		//使用技能结果 广播
  SKILL_USE_BROAD_RSP = 704;			//生物使用技能 广播
  SKILL_WARN_BROAD_RSP = 705;			//技能预警 广播
  SKILL_CANCEL_USE_REQ	= 706;		//请求 取消使用技能
  SKILL_CANCEL_USE_BROAD_RSP = 707;	//生物取消使用技能 广播
  SKILL_GROUP_UPDATE_RSP = 708;		//技能组更新
  SKILL_INFO_REQ = 709;				//请求技能详细信息
  SKILL_INFO_RSP = 710;				//返回技能详细信息
  SKILL_INFO_UPDATE_RSP = 711;		//技能详细信息更新
  SKILL_GROUP_CHG_REQ = 712;			//请求切换技能组
  SKILL_GROUP_CHG_RSP = 713;			//返回 切换技能组 结果
  SKILL_INFO_DEL_RSP = 714;			//返回 移除技能
  SKILL_WAKE_UP_REQ = 715;			//请求技能觉醒
  SKILL_WAKE_UP_RSP = 716;			//返回技能觉醒结果
  SKILL_CHG_POS_REQ = 717;			//请求改变技能位置
  SKILL_CHG_POS_RSP = 718;			//返回改变技能位置结果



  //buff 751-800
  BUFF_LIST_REQ = 751;			//请求 buff列表
  BUFF_LIST_RSP = 752;			//返回 buff列表
  BUFF_UPDATE_RSP = 753;			//更新 buff信息
  BUFF_DEL_RSP = 754;				//移除 buff
  BUFF_EFFECT_BROAD_RSP = 755;	//buff效果 广播


  // boss 信息
  NOTIFY_BOSS_SIMPLE_RSP = 850;	//BOSS简单数据
  BOSS_DATA_REQ	= 851;			//boss信息请求
  BOSS_DATA_RSP   = 852;			//boss信息返回
  NOTIFY_SCENE_BOSS_DATA_RSP = 853;	//进入场景BOSS数据请求
  NOTIFY_SCENE_BOSS_UPDATE_RSP = 854;	//BOSS 场景更新
  NOTIFY_BOSS_INFO_RSP	= 855;		//BOSS 信息更新
  ENTER_BOSS_MAP_REQ		= 856;	 	//5-1进入BOSS地图
  ENTER_BOSS_MAP_RSP		= 857;		//进入BOSS地图返回
  ATTENT_BOSS_REQ	= 858; 				//关注Boss请求
  ATTENT_BOSS_RSP = 859;				//关注BOSS返回
  BOSS_DROP_REQ = 860;			//boss掉落
  BOSS_DROP_RSP = 861;			//boss掉落
  NOTIFY_BOSS_RESULT_RSP = 862;	//boss结算
  BOSS_KILL_REQ = 863;			//快速击杀boss
  BOSS_KILL_RSP = 864;			//快速击杀boss返回
  //外关模块 900 - 950
  CLIENT_TO_LOGIC_FACADE_BEGIN					= 900;
  CLIENT_TO_LOGIC_FACADE_INFO_REQ					= 901;//外观信息
  LOGIC_TO_CLIENT_FACADE_INFO_RSP					= 902;//外观信息
  CLIENT_TO_LOGIC_FACADE_LEVUP_REQ				= 903;//外观升级请求
  LOGIC_TO_CLIENT_FACADE_LEVUP_RSP				= 904;//外观升级
  CLIENT_TO_LOGIC_FACADE_DRESS_REQ				= 905;//外观穿戴
  LOGIC_TO_CLIENT_FACADE_DRESS_RSP				= 906;//外观穿戴
  CLIENT_TO_LOGIC_FACADE_UNDRESS_REQ				= 911;//外观重置
  LOGIC_TO_CLIENT_FACADE_UNDRESS_RSP				= 912;//外观重置
  CLIENT_TO_LOGIC_FACADE_FRAGEMNT_USE_REQ	= 913;//外观(羽灵)使用
  LOGIC_TO_CLIENT_FACADE_FRAGEMNT_USE_RSP	= 914;//外观(羽灵)使用

  CLIENT_TO_LOGIC_FACADE_SKILL_UNLOCK_NOTIFY = 915; //外观技能解锁
  CLIENT_TO_LOGIC_FACADE_SKILL_LEV_REQ = 916; //技能升级
  LOGIC_TO_CLIENT_FACADE_SKILL_LEV_RSP = 917; //技能升级

  CLIENT_TO_LOGIC_FACADE_FANTASY_INFO_REQ		= 920;//外观幻化信息
  LOGIC_TO_CLIENT_FACADE_FANTASY_INFO_RSP		= 921;//外观幻化信息
  CLIENT_TO_LOGIC_FACADE_FANTASY_ACTIVE_REQ	= 922;//外观幻化激活
  LOGIC_TO_CLIENT_FACADE_FANTASY_ACTIVE_RSP	= 923;//外观幻化激活
  CLIENT_TO_LOGIC_FACADE_FANTASY_LEVELUP_REQ = 924;//外观幻化升级
  LOGIC_TO_CLIENT_FACADE_FANTASY_LEVELUP_RSP = 925;//外观幻化升级
  CLIENT_TO_LOGIC_FACADE_FANTASY_DRESS_REQ   = 926;//外观幻化穿戴
  LOGIC_TO_CLIENT_FACADE_FANTASY_DRESS_RSP   = 927;//外观幻化穿戴
  CLIENT_TO_LOGIC_FACADE_FANTASY_UNDRESS_REQ	= 928;//外观幻化重置
  LOGIC_TO_CLIENT_FACADE_FANTASY_UNDRESS_RSP	= 929;//外观幻化重置

  CLIENT_TO_LOGIC_FACADE_END					= 950;

  //mini activity 951 - 999
  CLIENT_TO_LOGIC_MINI_ACTIVITY_BEGIN = 951;
  //god relics
  CLIENT_TO_LOGIC_GET_GOD_RELICS_INFO_REQ = 952; //获得天机谱活动数据
  LOGIC_TO_CLIENT_GET_GOD_RELICS_INFO_RSP = 953; //获得天机谱活动数据

  LOGIC_TO_CLIENT_UPDATE_GOD_RELICS_NOTIFY = 954; //当前任务更新, 小任务完成了，或整个任务组完成了

  CLIENT_TO_LOGIC_GET_GOD_RELICS_TASK_REWARD_REQ = 955; //领取小任务奖励
  LOGIC_TO_CLIENT_GET_GOD_RELICS_TASK_REWARD_RSP = 956; //领取小任务奖励

  CLIENT_TO_LOGIC_GET_GOD_RELICS_GROUP_TASK_REWARD_REQ = 957; //领取任务组奖励，以及通知新的任务组
  LOGIC_TO_CLIENT_GET_GOD_RELICS_GROUP_TASK_REWARD_RSP = 958; //领取任务组奖励，以及通知新的任务组

  //dail task
  CLIENT_TO_LOGIC_GET_DAILY_TASK_DATA_REQ = 960; //获得Daily活动数据
  LOGIC_TO_CLIENT_GET_DAILY_TASK_DATA_RSP = 961; //获得Daily活动数据

  LOGIC_TO_CLIENT_UPDATE_DAILY_TASK_NOTIFY = 962; //任务更新, 任务完成，解锁，状态改变 奖励状态更新

  CLIENT_TO_LOGIC_GET_DAILY_TASK_REWARD_REQ = 964; //请求每日任务奖励
  LOGIC_TO_CLIENT_GET_DAILY_TASK_REWARD_RSP = 965; //请求每日任务奖励

  CLIENT_TO_LOGIC_DAILY_TASK_LEVELUP_REQ = 966; //每日任务养成等级提升
  LOGIC_TO_CLIENT_DAILY_TASK_LEVELUP_RSP = 967; //每日任务养成等级提升

  CLIENT_TO_LOGIC_DAILY_LIMIT_TASK_ADD_READY_REQ = 968; //限时任务投名
  LOGIC_TO_CLIENT_DAILY_LIMIT_TASK_ADD_READY_RSP = 969; //限时任务投名

  CLIENT_TO_LOGIC_DAILY_BACK_TASK_BACK_REQ = 970; //找回请求
  LOGIC_TO_CLIENT_DAILY_BACK_TASK_BACK_RSP = 971; //找回请求

  CLIENT_TO_LOGIC_DAILY_LIMIT_TASK_ADD_ALL_READY_REQ = 972; //限时任务投名
  LOGIC_TO_CLIENT_DAILY_LIMIT_TASK_ADD_ALL_READY_RSP = 973; //限时任务投名

  CLIENT_TO_LOGIC_DAILY_BACK_ALL_TASK_BACK_REQ = 974; //找回请求
  LOGIC_TO_CLIENT_DAILY_BACK_ALL_TASK_BACK_RSP = 975; //找回请求

  CLIENT_TO_LOGIC_MINI_ACTIVITY_END = 999;

  //副本1000-1200
  ENTER_DUP_REQ	= 1000;			//进入副本请求
  ENTER_DUP_RSP 	= 1001;			//进入副本返回
  DUP_RESULT_REQ	= 1002;			//请求副本结果
  DUP_RESULT_RSP	= 1003;			//请求副本结果返回
  DUP_LEAVE_REQ 	= 1004;			//离开副本请求
  DUP_LEAVE_RSP	= 1005;			//离开副本返回
  NOTIFY_DUP_END_RSP = 1006;		//副本已结束
  NOTIFY_DUP_RSP	= 1007;			//上线通知副本数据及副本数据更新
  NOTIFY_DUP_RUN_RSP = 1008;		//副本运行中数据更新
  NOTIFY_DUP_BEGIN_CD_RSP = 1009;	//玩家进入副本，战前倒计时
  RECV_TOWER_DUP_REQ = 1010;		//领取爬塔奖励
  RECV_TOWER_DUP_RSP = 1011;		//领取爬塔奖励返回
  BUY_DUP_NUM_REQ	 = 1012;		//购买副本次数请求
  BUY_DUP_NUM_RSP  = 1013;		//购买副本次数返回
  DUP_TOWER_RECORD_REQ = 1014;	//查看爬塔日志记录
  DUP_TOWER_RECORD_RSP = 1015;	//查看爬塔日志记录返回


  //活动 1201 - 1300
  ACT_LIST_REQ = 1201;			//请求活动列表
  ACT_LIST_RSP = 1202;			//返回活动列表

  //组队
  CLIENT_TO_SERVER_TEAM_BEGIN				= 1300;
  CLIENT_TO_CENTER_CREATE_TEAM			= 1301;	//请求创建队伍
  CLIENT_TO_CENTER_APPLY_JOIN_TEAM		= 1302;	//申请加入队伍
  CLIENT_TO_CENTER_REPLY_JOIN_TEAM		= 1303;	//回复加入队伍
  CLIENT_TO_CENTER_QUIT_TEAM				= 1304;	//退出队伍
  CLIENT_TO_CENTER_INVITE_JOIN_TEAM		= 1305;	//邀请入队
  CLIENT_TO_CENTER_REPLY_INVITE_JOIN_TEAM	= 1306;	//回复邀请入队
  CLIENT_TO_CENTER_KICK_TEAM_MEMBER		= 1307;	//请离队员
  CLIENT_TO_CENTER_TEAM_APPLY_LIST		= 1308;	//获取申请列表
  CLIENT_TO_CENTER_CLEAR_APPLY_LIST		= 1309;	//清空申请列表
  CLIENT_TO_CENTER_TEAM_SET_TARGET		= 1310;	//设置队伍目标
  CLIENT_TO_CENTER_TEAM_PROMOTE_LEADER	= 1311;	//提升为队长
  CLIENT_TO_CENTER_TEAM_AUTO_MATCH		= 1312;	//自动匹配
  CLIENT_TO_SERVER_TEAM_END				= 1320;

  SERVER_TO_CLIENT_TEAM_BEGIN				= 1330;
  CENTER_TO_CLIENT_CREATE_TEAM			= 1331;	//请求创建队伍
  CENTER_TO_CLIENT_APPLY_JOIN_TEAM		= 1332;	//申请加入队伍
  CENTER_TO_CLIENT_REPLY_JOIN_TEAM		= 1333;	//回复加入队伍
  CENTER_TO_CLIENT_QUIT_TEAM				= 1334;	//退出队伍
  CENTER_TO_CLIENT_INVITE_JOIN_TEAM		= 1335;	//邀请入队
  CENTER_TO_CLIENT_REPLY_INVITE_JOIN_TEAM	= 1336;	//回复邀请入队
  CENTER_TO_CLIENT_KICK_TEAM_MEMBER		= 1337;	//请离队员
  CENTER_TO_CLIENT_TEAM_APPLY_LIST		= 1338;	//获取申请列表
  CENTER_TO_CLIENT_CLEAR_APPLY_LIST		= 1339;	//清空申请列表
  CENTER_TO_CLIENT_TEAM_SET_TARGET		= 1340;	//设置队伍目标
  CENTER_TO_CLIENT_TEAM_PROMOTE_LEADER	= 1341;	//提升为队长
  CENTER_TO_CLIENT_TEAM_AUTO_MATCH		= 1342;	//自动匹配
  CENTER_TO_CLIENT_UPDATE_TEAM_APPLY		= 1343;	//更新申请列表通知
  CENTER_TO_CLIENT_UPDATE_TEAM_MEM		= 1344;	//更新队伍成员信息
  CENTER_TO_CLIENT_UPDATE_TEAM_MEM_SIMPLE	= 1345;	//更新队伍成员简要信息
  CENTER_TO_CLIENT_INVITE_TEAM_NOTIFY		= 1346;	//邀请加入队伍通知
  CENTER_TO_CLIENT_UPDATE_TEAM_INFO		= 1347;	//更新队伍信息
  CENTER_TO_CLIENT_UPDATE_TEAM_SIMPLE		= 1348;	//更新队伍简要信息
  CENTER_TO_CLIENT_TEAM_REFUSE_NOTIFY		= 1349;	//队伍拒绝通知
  SERVER_TO_CLIENT_TEAM_END				= 1350;

  SOCIAL_TO_CENTER_CREATE_TEAM			= 1360;	//社交服通知中心服创建队伍
  SOCIAL_TO_CENTER_CHAR_JOIN_TEAM			= 1361;	//社交服通知中心服玩家加入队伍
  SOCIAL_TO_CENTER_CHAR_QUIT_TEAM			= 1362;	//社交服通知中心服玩家加入队伍
  SOCIAL_TO_CENTER_TEAM_LEADER			= 1363;	//社交服通知中心服队长改变
  SOCIAL_TO_CENTER_TEAM_DISMISS			= 1364;	//社交服通知中心服队伍解散
  SOCIAL_TO_CENTER_TEAM_DUP_BEGIN			= 1365;	//社交服通知中心服 队伍副本开始
  SOCIAL_TO_CENTER_TEAM_DUP_END			= 1366;	//社交服通知中心服 队伍副本结束
  SOCIAL_TO_CENTER_TEAM_MEM_UPDATE		= 1367;	//社交服通知中心服队伍成员信息更新

  CENTER_TO_LOGIC_UPDATE_TEAM_NOTIFY		= 1368;	//组队信息更新
  CENTER_TO_LOGIC_DISMISS_TEAM_NOTIFY		= 1369;	//中心服通知逻辑服 队伍解散
  CENTER_TO_LOGIC_UPDATE_TEAM_CLONESS_NOTIFY = 1370;	//中心服通知逻辑服 玩家队伍亲密度等级
  CENTER_TO_LOGIC_TEAM_INFO_NOTIFY		= 1371;	//中心服通知逻辑服 队伍信息

  LOGIC_TO_SOCIAL_TEAM_ROBOT_HP_SYNC		= 1372;	//逻辑服组队机器人同步血量到社交服
  LOGIC_TO_SOCIAL_TEAM_ENTER_NEXT_PALACE_LAYER = 1373;	//逻辑服请求社交服队伍进入下一层工会地宫副本
  LOGIC_TO_CLIENT_DUPLICATE_CANCEL_STATE			= 1375;	//取消组队状态通知

  //掉落 1450-1500
  DROP_PICK_REQ = 1450;			//请求拾取掉落
  DROP_PICK_RSP = 1451;			//返回 拾取掉落结果
  DROP_HEAD_NOTIFY = 1452;		//玩家头顶 掉落归属

  //vip
  NOTIFY_VIP_DATA_RSP = 1480;		//vip 经验

  CLIENT_SERVER_MAX_CMD = 5000;	//最大协议ID
};	